testSuite/scripts/utility.py

```diff
@@ -25,6 +25,16 @@ def add_arguments(self, argument):
         if argument == None:
             return
         self.args.append(argument)
+
+        # auto-set MD5 checking flags, so that we always check when testing
+        ct = str.lower(self.command_type)
+        is_copy_or_sync = ct == "copy" or ct == "cp" or ct == "sync"
+        if is_copy_or_sync and (not str.startswith(argument, "h")):  # this is a local location
```

Comment:
>(not str.startswith(argument, "h")) [](start = 31, length = 35)

I would be a little more clear about this... Perhaps check if it doesn't start with "https://"

Reply:
will change to http.  (Not https, in case we sometimes only use http)